python/ola/ClientWrapperTest.py

```diff
@@ -181,19 +182,32 @@ def d():
     self.start = datetime.datetime.now()
     wrapper.Run()
 
+    # Everything has been called
     self.assertIsNotNone(results.a_called)
     self.assertIsNotNone(results.b_called)
     self.assertIsNotNone(results.c_called)
     self.assertIsNotNone(results.d_called)
 
-    self.assertTrue(results.a_called - self.start <
-                    datetime.timedelta(milliseconds=5))
-    self.assertTrue(results.b_called - self.start >=
-                    datetime.timedelta(milliseconds=5))
-    self.assertTrue(results.c_called - self.start >=
-                    datetime.timedelta(milliseconds=10))
-    self.assertTrue(results.d_called - self.start >=
-                    datetime.timedelta(milliseconds=15))
+    # Check when the callbacks were called. Allow 500 microseconds of drift.
+    # Called immediately
+    a_diff = results.a_called - self.start
+    self.assertLessEqual(a_diff,
+                         datetime.timedelta(microseconds=500))
```

Comment:
It looks like https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertAlmostEqual and feeding in a delta should work and will likely give some far more friendly diagnostics, given if it fails you actually want to know the two numeric values being compared, not the delta.

Untested, but basically this:
```suggestion
    a_diff = results.a_called - self.start
    self.assertAlmostEqual(results.a_called - self.start, 0, None, None, timedelta(microseconds=500))
```

And equivalent for the others.